src/Api/V1/Init.php

```diff
@@ -59,17 +59,17 @@ public function initialize($type)
      */
     private function definePaths()
     {
-        define('API_CORE_PATH', PATH_WWW . '/' . APPLICATION);
-        define('BACKEND_PATH', PATH_WWW . '/src/Backend');
-        define('BACKEND_CACHE_PATH', BACKEND_PATH . '/Cache');
-        define('BACKEND_CORE_PATH', BACKEND_PATH . '/Core');
-        define('BACKEND_MODULES_PATH', BACKEND_PATH . '/Modules');
-
-        define('FRONTEND_PATH', PATH_WWW . '/src/Frontend');
-        define('FRONTEND_CACHE_PATH', FRONTEND_PATH . '/Cache');
-        define('FRONTEND_CORE_PATH', FRONTEND_PATH . '/Core');
-        define('FRONTEND_MODULES_PATH', FRONTEND_PATH . '/Modules');
-        define('FRONTEND_FILES_PATH', FRONTEND_PATH . '/Files');
+        defined('API_CORE_PATH') || define('API_CORE_PATH', PATH_WWW . '/' . APPLICATION);
```

Comment:
We should think about putting these in one place, since this is the same in the frontend as in the backend.

I don't really think these are the constants we should remove the fastest, since they are always the same in every Fork application. (so they are really constant)